RELEASE.md

```diff
@@ -52,5 +54,6 @@ released version of Polkadot, without taking *every* change to `master` since
 the last release. For example, in the event of a security vulnerability being
 found, where releasing a fixed version is a matter of some expediency. In cases
 like this, the fix should first be merged with master, cherry-picked to a branch
-forked from `release`, tested, and then finally merged with `release`. A
-sensible versioning scheme for changes like this is `vX.Y.Z-1`.
+forked from the last release-candidate branch, tested, and then finally merged
+with the release-candidate branch. A sensible versioning scheme for changes like
+this is `vX.Y.Z-1`, though this can cause problems with the RPM packaging.
```

Comment:
So we remain with this potential issue or do you think we can work around ?
If we expect problems, we should point to a solutioN/workaround.